editor/debugger/script_editor_debugger.cpp

```diff
@@ -901,6 +904,13 @@ void ScriptEditorDebugger::_breakpoint_tree_clicked() {
 	}
 }
 
+String ScriptEditorDebugger::_format_frame_text(const ScriptLanguage::StackInfo *info) {
+	String text = info->file.get_file() + ":" + itos(info->line) + " @ " + info->func;
+	if (!text.ends_with(")"))
+		text += "()";
```

Comment:
We mandate braces even around single line `if` statements in C++ (not in C# I think).